### ISSUE TITLE
Error retrieving game data.

### ISSUE THREAD

**Issue by ScytheandGenesisGames** [Project-Lunar/Project-Lunar-Issue-Tracker#42]
it displays this

![image](https://user-images.githubusercontent.com/2251421/109472862-a8000000-7a40-11eb-933e-3dd6c6517c80.png)

**Comment by IvanJMembrilla** [comment]
The same problem for me :(

Someone told that running as admin solves the problem. For me didn't take effect. Could you try this way and see if you're lucky.

**Comment by testycaracal** [comment]
Same here.

The "run as admin" solution doesn't work.

Any update about this issue ?

**Comment by IvanJMembrilla** [comment]
In my case, I realised that you can choose between two data servers for get the game information. The first one doesn't works, but the second works without problems.

You should also understand that the game name is the field that is used by the server to search the information so I recommend to rewrite the name before do the search.

For example: "Lion king, the".

I founded many roms with this type of name that, obviously, doesn't find results in the searching. If you rewrite the name just like "Lion", the server will find all the games with the word "lion" and then you just need to select the proper result.

That's all.

I hope all of you can resolve your problems with this.

Good luck!

**Comment by testycaracal** [comment]
Yes TheGamesDB doesn't work anymore but ScreenScraper should but I always get the same error (404 not found), no matter what I use as a search term.

**Comment by Cordova82** [comment]
Is this problem solved? I having the same problem

**Comment by studiostriver** [comment]
I getting the same problem after i tried to instal my first game, with every i tried having the same issue. Where you have option to choose second data server? I tried finding them but no luck so far. :(

**Comment by IvanJMembrilla** [comment]
Try selecting the other server on "Scraper source", just below the button "Get game information". The one which is selected by default with the name "ScreenScraper" seems to be inoperative, so select the other one.

**Comment by studiostriver** [comment]
> 
> 
> Try selecting the other server on "Scraper source", just below the button "Get game information". The one which is selected by default with the name "ScreenScraper" seems to be inoperative, so select the other one.

It worked, and i was starting to put games in the system, but after i put some games screen just stopped sending signal and it blank. :(
When i connect the software it see the console and it saying its connected, but i getting no video signal out of my hdmi. This is terrible. :( I really feel down.

**Comment by IvanJMembrilla** [comment]
> > Try selecting the other server on "Scraper source", just below the button "Get game information". The one which is selected by default with the name "ScreenScraper" seems to be inoperative, so select the other one.
> 
> It worked, and i was starting to put games in the system, but after i put some games screen just stopped sending signal and it blank. :(
> When i connect the software it see the console and it saying its connected, but i getting no video signal out of my hdmi. This is terrible. :( I really feel down.

Try to restore the security copy that you should have saved before to install it and if this fix the problem start with all the process again.

**Comment by studiostriver** [comment]
> 
> 
> > > Try selecting the other server on "Scraper source", just below the button "Get game information". The one which is selected by default with the name "ScreenScraper" seems to be inoperative, so select the other one.
> > 
> > 
> > It worked, and i was starting to put games in the system, but after i put some games screen just stopped sending signal and it blank. :(
> > When i connect the software it see the console and it saying its connected, but i getting no video signal out of my hdmi. This is terrible. :( I really feel down.
> 
> Try to restore the security copy that you should have saved before to install it and if this fix the problem start with all the process again.

After trail and errors i finally fixed things, i restored everything, and it still didnt want to show the video signal, for some reason after i unplugged all cable and plugged again, and finally from that point it started working.
